packages/flutter/lib/src/widgets/navigator.dart

```diff
@@ -80,9 +80,11 @@ abstract class Route<T> {
   /// is replaced, or if the navigator itself is disposed).
   void dispose() { }
 
-  // If the route's transition can be popped via a user gesture (e.g. the iOS
-  // back gesture), this should return a controller object that can be used
-  // to control the transition animation's progress.
+  /// If the route's transition can be popped via a user gesture (e.g. the iOS
+  /// back gesture), this should return a controller object that can be used
+  /// to control the transition animation's progress.
+  ///
+  /// Otherwise, it should return null.
```

Comment:
Does this mandate a new paragraph? It seems to be a pretty good continuation of the first sentence.